### ISSUE TITLE
Reports via website do not reach DWG

### ISSUE THREAD

**Issue by ivanbranco** [openstreetmap/openstreetmap-website#4568]
### URL

https://www.openstreetmap.org/user/username

### How to reproduce the issue?

1. Go to username page
2. Click "Report this user"
3. Choose a reason, write the report and send clicking "Create report"
4. I often don't get an automated reply and the report doesn't reach DWG. A DWG member [confirmed it](https://community.openstreetmap.org/t/odd-edits-in-italy/97706/89) in September. Last time was 1-2 days ago, I reported a user and again no automated reply and not manual reply for DWG. I would say that for 1/3 of the reports I write I don't get a reply at all (manual or automatic).

### Screenshot(s) or anything else?

I use Google Chrome + Windows 11 Home

**Comment by tomhughes** [comment]
Whether a report goes to the moderators (aka DWG) or the administrators (mostly me) will depend on what options are chosen when the report is made.

There are no automatic acknowledgements for any reports at all - if a report goes to DWG then they might open a ticket in OTRS and that might send an acknowledgement.

Anything which is handed by be will not receive any response unless I need more information.

**Comment by ivanbranco** [comment]
> Whether a report goes to the moderators (aka DWG) or the administrators (mostly me) will depend on what options are chosen when the report is made.

I guess the first 4th options (spam/offensive/threats/vandal) go to DWG and the fifth option (other) goes to administrators?
I often choose "other" because vandal is an overstatment most of times (maybe is just a mapper in good faith which is not responsive to changeset comments because email is broken ecc.). But if this is the way to reach DWG I will start using the "vandal" option.

**Comment by tomhughes** [comment]
The logic is at https://github.com/openstreetmap/openstreetmap-website/blob/56570cabf19b73cb496f7482c87bda0648e66ba6/app/controllers/reports_controller.rb#L57 and only note reports and vandal reports for users go direct to DWG.

That said I will move other reports to DWG if necessary but as I have no idea which reports exactly you think have "gone missing" I can't say exactly what happened to them.

**Comment by ivanbranco** [comment]
I wrote a new ticket using the "This user is a vandal" option this time, March 9 around 13:30 GTM+1. I got no automatic replies nor manual replies so far.
For example I saw that this user is only two days old and already blocked: https://www.openstreetmap.org/user/Land%20Owner
I still don't understand if my reports via website are not delivered or if I'm being ignored. I read in another issue that it can happen: 

> The are a couple of exceptions to this - some reports can just be ignored (e.g. obviously not in good faith)

but I don't think is my case.
